docs/user-guide/troubleshooting.md

```diff
@@ -8,16 +8,54 @@ Do not move it without providing redirects.
 
 # Troubleshooting
 
-## Authentication to the Kubernetes API Server
-If your Kubernetes cluster is not configured correctly, it may fail to contact
-the API server. One way this manifests is when users attempt to connect to the
-UI in their web browser and see a message like this:
 
-    Get https://1.2.3.4/api/v1/replicationcontrollers: x509: failed to load system roots and no roots provided
+## Authentication to the Kubernetes API Server
 
-This means that Dashboard failed to authenticate to the API server. Before
+If your Kubernetes cluster is not configured correctly, it may fail to contact
+the API server. Before
 explaining the solution, it is useful to review how the dashboard discovers
 and authenticates with the API server.
+A number of components are involved in the authentication process and the first step is narrow
+down the source of the problem, mainly if you have problems with user authentication
```

Comment:
...problem, **namely whether it is a problem with user authentication or with service authentication**.